cli/internal/cmd/create.go

```diff
@@ -209,6 +210,27 @@ func checkDirClean(fileHandler file.Handler) error {
 	return nil
 }
 
+func translateCreateErrors(cmd *cobra.Command, err error) error {
+	switch err {
+	case terraform.ErrTerraformWorkspaceDifferentFiles:
+		cmd.PrintErrln("\nYour current working directory contains an existing Terraform workspace which does not match the expected state.")
+		cmd.PrintErrln("This can be due to a version mix up or a corrupted workspace.")
+		cmd.PrintErrln("Before creating a new cluster, try \"constellation terminate\".")
+		cmd.PrintErrf("If this does not work, either move or delete the directory \"%s\".\n", constants.TerraformWorkingDir)
+		cmd.PrintErrln("Please only delete the directory if you made sure that all created cloud resources have been terminated.")
+		return err
+	case terraform.ErrTerraformWorkspaceExistsWithDifferentVariables:
+		cmd.PrintErrln("\nYour current working directory contains an existing Terraform workspace which was initiated with different input variables.")
+		cmd.PrintErrln("This can be the case if you have tried to create a cluster before with different options which did not complete, or the workspace is corrupted.")
+		cmd.PrintErrln("Before creating a new cluster, try \"constellation terminate\".")
+		cmd.PrintErrf("If this does not work, either move or delete the directory \"%s\".\n", constants.TerraformWorkingDir)
```

Comment:
```suggestion
		cmd.PrintErrf("If this does not work, either move or delete the directory %q.\n", constants.TerraformWorkingDir)
		cmd.PrintErrln("Please only delete the directory if you made sure that all created cloud resources have been terminated.")
		return err
	case terraform.ErrTerraformWorkspaceExistsWithDifferentVariables:
		cmd.PrintErrln("\nYour current working directory contains an existing Terraform workspace which was initiated with different input variables.")
		cmd.PrintErrln("This can be the case if you have tried to create a cluster before with different options which did not complete, or the workspace is corrupted.")
		cmd.PrintErrln("Before creating a new cluster, try \"constellation terminate\".")
		cmd.PrintErrf("If this does not work, either move or delete the directory %q.\n", constants.TerraformWorkingDir)
```